src/main/java/com/solvd/micro9/scooters/service/KStreamsService.java

```diff
@@ -0,0 +1,9 @@
+package com.solvd.micro9.scooters.service;
+
+import java.math.BigDecimal;
+
+public interface KStreamsService {
```

Comment:
the interface name is quite misleading

using such an interface I assume that it has all methods related to the Stream Processing logic, a.k.a god object
so, by naming your interface like this you inevitably end up with the good object, obvious that it is _bad design_

@eyakauleva wdyt?